OpenSim/Common/Exception.h

```diff
@@ -67,14 +110,31 @@ class OSIMCOMMON_API Exception  : public std::exception {
     std::string _file;
     /** Line number at which the error occurred. */
     int _line;
+    /** Delimiter/separator used to separate different parts of the message.  */
+    static const std::string _msgSeparator;
```

Comment:
Why is this separator parameterized? I don't see any way to change it. This could be generalized later if needed, once we have a use case. Or is there a current use case that I'm missing?